fxmanifest.lua

```diff
@@ -1,23 +1,24 @@
-fx_version "cerulean"
+fx_version("cerulean")
 
-lua54 "yes"
+lua54("yes")
 
-game "gta5"
+game("gta5")
 
```

Comment:
is this your formatter changing these? This doesnt really conform to standard fivem conventions, I'm fine with the changes, just wondering why

Reply:
Yeah, these are annoying nvim changes. I'll revert them.